docs/source/mixed-precision.rst

```diff
@@ -3,39 +3,67 @@ Mixed precision training
 
 In this section we will describe
 
-* How to train existing models in mixed precision
-  (basically, two lines change in the config: dtype and loss scaling).
+* How to train models in mixed precisionfloat for training on Volta GPUs
 
-* How mixed precision is implemented in our toolkit (optimizer and regularizer
-  wrappers)
+* How to port existing models from float32 to mixed precisionfloat
+  
+* How mixed precision is implemented in our toolkit 
 
-* What are the general steps needed to add mixed precision support to a new
-  model. Should also contain example of what we did for our cases: profiling
-  GPU/CPU placement, profiling which operations are bottleneck,
-  debugging infs/nans, converting some layers to fp32, etc.
+Mixed precision training on Volta GPUs
+--------------------------
 
-Why mixed precision
--------------------
+Half-precision floating numbers (float16) have limited numerical range compared to single-precision float (float32). This constarined numerical range can lead to overflow or underflow during training. We use two techniques to prevent these hazards. First, we  maintain a "master" float32 copy of the weights that accumulates the gradients after each optimizer step.  Second, we use global scaling applied to the loss to prevent the loss of information during backpropagation. We demonstrated that this approach for GNMT(translation) and DeepSpeech2 (speech recognition).  Using this approach, we can reduce the memory consumption of deep learning models by nearly 2x. 
 
-... Should motivate the use of mixed precision: float-16 is great, does not
-always work, so using fp-32 weights and loss scaling. Cite the paper. ...
+For more details see "Mixed Precision Training" white paper https://arxiv.org/abs/1710.03740 
 
-How to use mixed precision
+How to port models from float32 to mixed precision
 --------------------------
-
 Enabling mixed precision with existing models in OpenSeq2Seq is easy:
-change ``dtype`` parameter of ``model_params`` dictionary to "mixed". You
-might need to enable loss scaling: either statically, by setting
+change ``dtype`` parameter of ``model_params`` to "mixed". 
+You might need to enable loss scaling: either statically, by setting
 ``loss_scale`` parameter inside ``model_params`` to the desired number, or
 you can use dynamic loss scaling by setting ``automatic_loss_scaling`` parameter
 to "Backoff" or "LogMax".
 
+To switch model from float32 to mixed precision, one should add dtype parameter to model description::
```

Comment:
I think we should have either top paragraph or what follows since they duplicate each other.